### ISSUE TITLE
document and use a consistent coding style

### ISSUE THREAD

**Issue by ddiss** [rapido-linux/rapido#121]
I propose that we follow blktests coding style:

- Indent with tabs.
- Don't add a space before the parentheses or a newline before the curly brace
  in function definitions.
- Variables set and used by the testing framework are in caps with underscores.
  E.g., TEST_NAME and GROUPS. Variables local to the test are lowercase
  with underscores.
- Functions defined by the testing framework or group scripts, including
  helpers, have a leading underscore. E.g., _have_scsi_debug. Functions local
  to the test should not have a leading underscore.
- Use the bash [[ ]] form of tests instead of [ ].
- Always quote variable expansions unless the variable is a number or inside of
  a [[ ]] test.
- Use the $() form of command substitution instead of backticks.
- Use bash for loops instead of seq. E.g., for ((i = 0; i < 10; i++)), not
  for i in $(seq 0 9).

**Comment by morbidrsa** [comment]
Good idea, I'd even propose we add a small .exrc and .dir-locals.el for this.
